en/docs/samples/oidc-token-encryption-sample.md

```diff
@@ -0,0 +1,196 @@
+# Encrypt ID Tokens
+
+This page guides you through configuring [token encryption for ID tokens](insertlink) using the WSO2 IS playground sample application. 
+
+----
+
+{!fragments/oauth-playground.md!}
+
+----
+
+## Enable ID token encryption
+
+1. On the management console, click **Service Providers > List** and **Edit** the "playground" service provider you created. 
+
+2. Expand **Inbound Authentication Configuration** and then **OAuth/OpenID Connect Configuration**. 
+
+3. Click **Edit**.
+
+4. Click **Enable ID Token Encryption** to enable id\_token encryption.
+
+    Once you enable id\_token encryption, two select boxes will be
+    visible to choose your preferred encryption algorithm and encryption
+    method.
+
+    -   **Encryption Algorithm:** Asymmetric encryption algorithm that is used to encrypt the Content Encryption Key (CEK), using the
+        public key of the service provider.
+
+    -   **Encryption Method:** Symmetric encryption algorithm that is used to encrypt the JWT claims set using the CEK.
+
+    Leave these values as they are if you do not have any specific requirements.
+
+-----
+
+{!fragments/encrypt-id-tokens.md!}
+
+----
+
+You have successfully set up token encryption and the sample Playground application. You can now try out one of the grant type flows to recieve an encrypted ID token and decrypt it.
+
+- [Authorization grant flow](#try-authorization-grant-flow)
+- [Implicit grant flow](#try-implicit-grant-flow)
+- [Password grant flow](#try-password-grant-flow)
+
+----
+
+## Try authorization grant flow
+
+1.  Visit the URL <http://wso2is.local:8080/playground2/oauth2.jsp> to start the application.
+
+2.  Enter the following details and click **Authorize**.
+
+    -   **Authorization Grant Type:** Authorization Code
+
+    -   **Client ID:** (the client ID received when you registered the service provider)
```

Comment:
when listing the other fields we have given the exact value to enter...since this is a description and not the exact value, I put it in brackets to differentiate.